### ISSUE TITLE
SSO button label goes out SSO button borders

### ISSUE THREAD

**Issue by antoniolgr** [bitwarden/desktop#575]
## Describe the Bug
The Italian label of the SSO button doesn't fit inside the SSO button borders

## Screenshots or Videos
<img width="1436" alt="Schermata 2020-10-27 alle 22 42 23" src="https://user-images.githubusercontent.com/48482600/97365412-ed030280-18a5-11eb-98cb-d3ed1be0b191.png">

## Environment

- Operating system: macOS 10.15.7
- Build Version: 1.22.2 (1.22.2)

**Comment by vachan-maker** [comment]
issue similar to #549 
Why can't this string be replaced by Enterprise SSO instead of Enterprise Single Sign On
@cscharf

**Comment by kspearrin** [comment]
I don't think SSO is universally understood outside of the developer world.

**Comment by mind-overflow** [comment]
Why don't we just allow the button to wrap the text and make a new line instead of trying to find ways to shorten the text? This would be a definitive and future-proof solution, considering that we don't know if any other language might have (now or in the future) similar issues.

**Comment by cscharf** [comment]
> Why don't we just allow the button to wrap the text and make a new line instead of trying to find ways to shorten the text?

Yes, that is the correct answer; the button should flex/wrap accordingly, while maintaining spacing, scaling and functional coherence. This will likely be the solution to this issue (and other text overflow issues on buttons where appropriate).

**Comment by vachan-maker** [comment]
> 
> 
> I don't think SSO is universally understood outside of the developer world.

Zoom has a login with sso feature and they use the string "Login with SSO". I think users will be fine with it. Just a suggestion, thanks

**Comment by vachan-maker** [comment]
On the desktop app
![image](https://user-images.githubusercontent.com/65799568/152284823-bdaa2cf8-ed2a-4715-b8fa-d7750f506de8.png)
On the web app
![image](https://user-images.githubusercontent.com/65799568/152284853-8facb0c5-7d2b-4d12-8e65-ad2b26d07e2c.png)
